Sources/TuistKit/Models/Platform.swift

```diff
@@ -43,3 +43,14 @@ extension Platform {
         }
     }
 }
+
+extension Platform {
+    var supportsLaunchScreen: Bool {
```

Comment:
Would you mind adding a documentation line to this one?

Reply:
Done